### PR TITLE
Skip all positions with ply less than or equal to 28

### PR DESCRIPTION
**Pull request by linrock**
This modification was used to train the net in https://github.com/official-stockfish/Stockfish/pull/4314

### CODE REVIEW

File: training_data_loader.cpp [hunk 1]
```diff
@@ -873,6 +873,9 @@ std::function<bool(const TrainingDataEntry&)> make_skip_predicate(bool filtered,
             if (e.score == VALUE_NONE)
                 return true;
 
+            if (e.ply <= 28)  // TODO enable setting ply count via a runtime flag
```
Comment (linrock) [inline training_data_loader.cpp:876]: maybe something like `--early-ply-skipping 28`

**Comment by Disservin** [comment]
Hi, 

I think I added that functionality <https://github.com/glinscott/nnue-pytorch/compare/master...Disservin:nnue-pytorch:param-skip_early_plies> would be cool if someone could test this since I have no idea how to run pytorch ^^

I can make a pr if you want or you can also copy the changes since its your idea anyway ; )

**Comment by linrock** [comment]
cool, feel free to open a PR since you've written the code already. easier to discuss the changes there

**Comment by linrock** [comment]
closing in favor of the configurable param approach in https://github.com/glinscott/nnue-pytorch/pull/222

**Pull request closed without merging**